index.js

```diff
@@ -1,11 +1,41 @@
-var oldHello = function(name) {
-  console.log('hello, ' + name + '!');
-};
+// var oldHello = function(name) {
+//   console.log('hello, ' + name + '!');
+// };
 
 
-const hello = (name) => {
-  console.log(`hello, ${name}!`);
-};
+// const hello = (name) => {
+//   console.log(`hello, ${name}!`);
+// };
 
-oldHello('world');
-hello('es6');
+// oldHello('world');
+// hello('es6');
+
+function printNames(){
+  var names = [ "Andy Mention",
+                "Emily Davis",
+                "Eric Fransen",
+                "Jessica Goulding",
+                "Jonmichael Chambers",
+                "Marc Garreau",
+                "Tan Doan",
+                "Alan Smith",
+                "Allison Larson",
+                "Andrew Watkins",
+                "Chad Brading",
+                "Corey Davis",
+                "Gustavo Villagrana",
+                "Hilary Denton",
+                "Horacio Chavez",
+                "Tim Proctor",
+                "Will Faurot" ]
+
+  var arrayLength = names.length;
+
+  for (var i = 0; i < arrayLength; i++) {
```

Comment:
This is a good solution here. You might also like `forEach` it'll feel more like Ruby, and less cumbersome than `for`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach 

Also, if you're wanting to play with es6, you can use `map`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map